src/main/scala/lms/thirdparty/cpu_array_ops/ArrayCPUOps.scala

```diff
@@ -67,6 +110,41 @@ trait ArrayCPUOps extends Dsl with ArrayOps {
     }
   }
 
+  def array_minus[T:Numeric:Manifest](a: Rep[Array[T]], b: Rep[Array[T]], res: Rep[Array[T]], size: Int)(implicit __pos: SourceContext) = {
```

Comment:
Would `size` be `Rep[Int]`? If `size` is `Int`, the loop doesn't have to iterate over a `Rep[Range]`.